zipkin-storage/elasticsearch/src/main/java/zipkin2/elasticsearch/internal/IndexNameFormatter.java

```diff
@@ -156,8 +156,8 @@ public List<String> formatTypeAndRange(@Nullable String type, long beginMillis,
   }
 
   String formatIndexPattern(String format, GregorianCalendar current, String prefix) {
-    return String.format(
-      format,
+    return 
+      format.formatted(
```

Comment:
I think this indent isn't intentional (move format.formatted back to previous line?)

Reply:
openrewrite had its own ideas on this one ;) will move it back, as yeah looks better